src/ssl.c

```diff
@@ -34894,12 +35575,9 @@ static int SetECKeyExternal(WOLFSSL_EC_KEY* eckey)
 
     return WOLFSSL_SUCCESS;
 }
-#endif /* HAVE_ECC && OPENSSL_EXTRA_X509_SMALL */
 
-#ifdef OPENSSL_EXTRA
-#ifdef HAVE_ECC
 /* EC_KEY Openssl -> WolfSSL */
-static int SetECKeyInternal(WOLFSSL_EC_KEY* eckey)
+int SetECKeyInternal(WOLFSSL_EC_KEY* eckey)
```

Comment:
Should still be static as far as I can tell.
Same for SetECKeyExternal.

Reply:
I use both in `wolfSSL_EVP_PKEY_derive`.

Reply:
I would like these functions to be in wolfcrypt/src/evp.c but am checking with Todd.

Reply:
Leave as is.